### ISSUE TITLE
Probably bug in Nexus

### ISSUE THREAD

**Issue by vcvrackideas** [JustMog/Mog-VCV#4]
Hi! Nexus is pretty cool kind of the divider! I found some strange thing - when gate connected like these - there is no light indication until I disconnect the subsequent output
![2020-03-01_19-22-33](https://user-images.githubusercontent.com/42435368/75629341-38b7bc00-5bf2-11ea-8b93-b45d0b269de0.png)

**Comment by JustMog** [comment]
Hi, this is intended. The light should flash only for the gates that correspond to that "stage" (in this case, the first two gates), which are sent to the top output.  
After that, the stage is "done" until reset is triggered, and does nothing except "pass gates on" - to the next stage by default, or to bottom output, if it's connected.  

You should only need to connect the bottom output if you want to do something like passing those gates to more than one other stage, anther instance of network, through a bernouli gate before the next stage, etc.
In short, try connecting only the top outputs of each stage for most uses. 

Hope this makes sense. Happy to help if you have any more questions

**Comment by JustMog** [comment]
To be clear, the module will only go to the next stage if the stages are connected to each other, either by default (when the bottom output isn't connected) or manually (when the bottom output is connected *to another stage's input*)
